tests/Unit/LoanServiceTest.php

```diff
@@ -276,7 +276,7 @@ public function testServiceCanRepayMultipleScheduledRepayments()
         $this->assertDatabaseHas('scheduled_repayments', [
             'id' => $scheduledRepaymentOne->id,
             'loan_id' => $loan->id,
-            'amount' => 1667,
```

Comment:
`amount` when creating the `$scheduledRepaymentOne` object was 1666: https://github.com/R4CHI7/laravel-code-challenge-2/blob/main/tests/Unit/LoanServiceTest.php#L236